input/pagecontent/reference.md

```diff
@@ -2776,7 +2788,7 @@ Following [standard profiling rules established in FHIR](https://www.hl7.org/fhi
 
 </div>
 
-* Restrict Observation.performer (nominally Reference(Practitioner | PractitionerRole | Organization | CareTeam | Patient | RelatedPerson) to allow only Practitioner:
+* Restrict Observation.performer (nominally Reference(Practitioner \| PractitionerRole \| Organization \| CareTeam \| Patient \| RelatedPerson)) to allow only Practitioner:
```

Comment:
Jekyll was formatting this line as a table since it saw `|` characters.  Escaping them seems to make Jekyll behave.